_code/InSituTree/R/supervisedSubcluster.R

```diff
@@ -0,0 +1,96 @@
+#' Supervised subclustering using a gene subset and insitutypeML
+#'
+#' @param reference_profiles Matrix of expression profiles, genes x cell types
+#' @param x Counts matrix, cells x genes
+#' @param neg Vector of mean negative controls for each cell
+#' @param quantile_absolute_expression_difference Minimum absolute expression
+#' difference within reference_profiles in terms of quantile level among all
+#' genes. Default = 0.5. Both quantile cutoffs must be passed to retain gene.
+#' @param quantile_percent_expression_difference Minimum percentage expression
+#' difference within reference_profiles in terms of quantile level among all
+#' genes. Default = 0.5. Both quantile cutoffs must be passed to retain gene.
+#' @param excluded_genes Genes to be excluded during fitting with InSituType
+#' @param cohort Vector of cells' cohort membership
+#' @param ... Additional parameters passed on to `insitutypeML`
```

Comment:
Sorry, my bad. I realized that `insitutypeML()` has option to do `protein` cell typing which is not considered in `InSituTree` and it does NOT have the extra control parameters on iterations and reference update (the only additional parameter of meaning for `InSituType` is `nb_size` which doesn't have much value to change). In this case, your original function without the flexibility of `...` additional parameters are a better option to reduce potential error.

Reply:
Yes, the only thing I thought `...` was adding was the `nb_size` parameter, which I don't typically use. I've re-removed these extra parameters.